frontrunner_sdk/commands/injective/get_orders.py

```diff
@@ -37,7 +37,7 @@ class GetOrdersRequest:
 
 @dataclass
 class GetOrdersResponse:
-  orders: Sequence[DerivativeOrderHistory]
+  orders: Sequence[DerivativeOrderHistoryWrapper]
```

Comment:
I think ours should take priority in naming, so we don't need the "Wrapper" distinction.

Ideally, sdk consumers would never actually need to reference the gRPC type; they would always access it through the "wrapper", so the type in signatures would always be the "wrapper".

I wonder if we can rename the existing "Order" class to "OrderCreate" and make this our order class. Or just drop the "Wrapper" part of the name and do...

```python
from pyinjective.proto.exchange.injective_derivative_exchange_rpc_pb2 import DerivativeOrderHistory as gRPC_DerivativeOrderHistory
```

or `_inj_DerivativeOrderHistory` or `_DerivativeOrderHistory` or any other variant.